apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/WalletTransfersFrontendIntegrationTest.scala

```diff
@@ -122,6 +122,10 @@ abstract class BaseWalletTransfersFrontendIntegrationTest
                 offerCard.childElement(className("transfer-offer-sender"))
               ) should matchText(expectedAns(aliceUserParty, aliceAnsName))
 
+              seleniumText(
+                offerCard.childElement(className("transfer-offer-received"))
+              ) should matchText(expectedAns(bobUserParty, bobAnsName))
+
```

Comment:
Does this pass? You shouldn't see both the sender and receiver at the same time according to the rendering logic below